src/appContext.js

```diff
@@ -1,13 +1,18 @@
-import React, { createContext } from 'react';
+import React, { createContext, useState } from 'react';
 
 const AppContext = createContext();
 
 const AppProvider = ({ children }) => {
-    return <AppContext.Provider value={{}}>{children}</AppContext.Provider>;
-};
+    const [pageResource, setPageResource] = useState({
+        mealPlans: [],
+        recipes: [],
+    });
```

Comment:
this file should look like you very first version (before this TICKET was created) so here we should only have AppProvider. thats all